tests/integration_tests/column_selector/run.sh

```
@@ -7,43 +7,79 @@ source $CUR/../_utils/test_prepare
 CDC_BINARY=cdc.test
 SINK_TYPE=$1
+CHANGEFEED_ID="test"
 
-function run() {
-	# test kafka sink only in this case
-	if [ "$SINK_TYPE" != "kafka" ]; then
-		return
-	fi
-
+function prepare_cluster() {
 	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)